log/server/logpipe.go

```diff
@@ -197,8 +197,8 @@ func (lp *LogPipe) Close(timeout time.Duration) error {
 // is stopped. It will close the `ready` channel you pass when ready to serve.
 func (lp *LogPipe) Boot(ready chan struct{}, db *sql.DB) error {
 	lp.connMutex.Lock()
-
-	lp.server = common.NewDBGRPCServer(db)
+	// LogPipe configuration for secure communication missing
+	lp.server = common.NewDBGRPCServer(true, db)
```

Comment:
What are the ramifications of setting this up without security?

Reply:
The first thing that comes to mind is that eavesdropping can occur if the logpipe daemon is outside the network or even on a local network. Data tampering may also be possible if I'm not mistaken, so a malicious actor could manipulate the log data.